src/RecordRowContext.js

```diff
@@ -0,0 +1,3 @@
+import { createContext } from 'react';
+
+export const RecordRowContext = createContext();
```

Comment:
I like the approach with context, just would like to suggest a few changes:
Name could be a bit more specific, maybe `SelectedAuthorityRecordContext`.

Also, it's a common practice to create two exports for one context file:
1. created context
2. context provider. You can create here a component `SelectedAuthorityRecordContextProvider` , move `useState` from `src/index` into this component and return `<SelectedAuthorityRecordContext.Provider>`

Also, please create `src/context` folder and move this file there

Reply:
Thank you for this note! Fixed now